includes/discovery/sensors/temperatures/linux.inc.php

```diff
@@ -15,3 +15,59 @@
         discover_sensor($valid['sensor'], 'temperature', $device, $sensor_oid, 1, $sensor_type, $descr, 1, 1, null, null, null, null, $value);
     }
 }
+
+echo 'HP_ILO ';
```

Comment:
It might be a good idea to retain the `if (startswith($sysObjectId, '.1.3.6.1.4.1.232.') ) {` check here.